app/uk/gov/hmrc/gform/testonly/TestOnlyFormService.scala

```diff
@@ -40,17 +51,64 @@ class TestOnlyFormService(
           val newId = java.util.UUID.randomUUID().toString
           cacheItem.data.fields.toList
             .traverse { case (k, v) =>
-              val value = if (k === "form") { v.as[JsObject] - "_id" - "userId" }
-              else v
-              snapshotRepository.put(newId)(DataKey(k), value)
+              if (k === "form") {
+                val formTemplateId = v.as[JsObject].value("formTemplateId").as[String]
+                val prefix = UniqueStringGenerator.generateUniqueString()
```

Comment:
Why not to use `newId` ?

Reply:
`newId` is too long to use as a prefix. Alex asked for something shorter. I came up with a way to encode the number of seconds since 1 jan 2024 as a alphanumeric string, so we have a new form id as for example `4xumt_tst123`

Reply:
So lets use first 8 (or 4 if you really insist) letters from `newId`.

Reply:
Potentially we might have a conflicts there. My method guarantees the uniqueness as long as it is not generated on the same second.

Reply:
and it is max 5-6 chars in the foreseeable future.